server.js

```diff
@@ -31,6 +32,12 @@ const out = (data, color = CWhite) => {
   if (!Array.isArray(data)) data = [data];
   process.stdout.write(color);
   process.stdout.write(getTime() + " ");
+
+  if (typeof data[0] === 'object') {
```

Comment:
can you give me an example of when does this happens?
And why don't we do this for all of the elements in data?

Reply:
It happens if you log an object, for example `console.log({foo: 'bar', bar: 'baz'})`.

In data[0] is the content of the first argument of the called function ('log' in this case). In data[1] would be the second parameter if string substitution is used.

Reply:
this seems like a very specific case and it makes the code inconsistent. for example, if I want to `console.log(object1, object2)` it'd display differently.

So I'd rather use the default console by nodejs.

One up-side of it is its colorized object by default.